homeassistant/components/honeywell/climate.py

```diff
@@ -116,7 +115,10 @@ async def async_setup_entry(hass, config, async_add_entities, discovery_info=Non
 
     data = hass.data[DOMAIN][config.entry_id]
 
-    async_add_entities([HoneywellUSThermostat(data, cool_away_temp, heat_away_temp)])
+    for device in data.devices:
+        async_add_entities(
+            [HoneywellUSThermostat(data, device, cool_away_temp, heat_away_temp)]
+        )
```

Comment:
```suggestion
    async_add_entities([
       HoneywellUSThermostat(data, device, cool_away_temp, heat_away_temp)
       for device in data.devices
    ])
```

Call `async_add_entities` once